src/components/base/button/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import propTypes from "prop-types";
+import styles from "components/base/button/index.module.scss";
+import { TYPE_SUBMIT } from "utils/const";
+
+function Button({ children, onButtonClick, className, buttonType }) {
```

Comment:
As the function is passed down as a prop and has the scope of the `<Button />` component, we can simply use `onClick` as function name. More concise with less overhead.
```suggestion
function Button({ children, onClick, className, buttonType }) {
```